spr/src/main.rs

```diff
@@ -0,0 +1,139 @@
+extern crate confy;
+
+#[macro_use]
+extern crate serde_derive;
+
+use std::fs;
+use std::io::{self, Write};
+use std::process::{Command, Stdio};
+
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Cli {
+    name: String,
+    #[structopt(short = "m", default_value = "run")]
+    mode: String,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+struct Config {
+    path: String,
+    src_dir: String,
+    out_dir: String,
+    test_dir: String,
+    test_ext: String,
+    file_ext: String,
+}
+
+impl ::std::default::Default for Config {
+    fn default() -> Self {
+        Self {
+            path: "/home/joseph/dev/codejam-js".to_string(),
```

Comment:
This should really panic the CLI. A path should be specified in a config file. The rest are all good defaults.